ebi_eva_common_pyutils/ena_utils.py

```diff
@@ -0,0 +1,33 @@
+import requests
+from lxml import etree
+
+
+def download_xml_from_ena(ena_url):
```

Comment:
Suggest using the usual retry decorator that we have used elsewhere for network calls...